src/mimetype.rs

```diff
@@ -1,11 +1,15 @@
-use std::{io, path::Path};
+use std::{fs, io, path::Path};
 
 /// image_or_video determines whether a given path is an image or video file
 pub fn image_or_video<P: AsRef<Path>>(filename: &P) -> Result<bool, io::Error> {
```

Comment:
Maybe we should change the name of the argument `filename` to `path` to denote that this path could be anything file or directory. What do you think?